### DISCUSSION TITLE
脚本失效，无窗体显示求问

### DISCUSSION THREAD

**Discussion by anyezhuguang** [the1812/Bilibili-Evolved#3627] [提问 / Q&A]
![image](https://user-images.githubusercontent.com/88629817/185971467-d408b207-f2f6-458c-82a4-b0cab4efed34.png)
重置过脚本可是仍然不显示窗体，关闭后正常，请问怎么办

**Comment by the1812** [comment]
去开个 Bug 反馈, 提供下必要的信息
![image](https://user-images.githubusercontent.com/26504152/187210813-cbc62aab-c671-4aa4-874b-5158c767fc2e.png)
